packages/xorg_server.rb

```diff
@@ -41,12 +50,12 @@ class Xorg_server < Package
   depends_on 'mesa'
 
   case ARCH
-    when 'armv7l', 'aarch64'
-      PEER_CMD_PREFIX='/lib/ld-linux-armhf.so.3'
-    when 'i686'
-      PEER_CMD_PREFIX='/lib/ld-linux-i686.so.2'
-    when 'x86_64'
-      PEER_CMD_PREFIX='/lib64/ld-linux-x86-64.so.2'
+  when 'armv7l', 'aarch64'
+    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'.freeze
+  when 'i686'
+    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'.freeze
+  when 'x86_64'
+    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'.freeze
```

Comment:
Why the `.freeze`?

Reply:
rubocop... is it bad?

Reply:
Just not understanding what needs to be "frozen".

Reply:
```
Offenses:

forupload/xorg_server.rb:54:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
forupload/xorg_server.rb:57:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
forupload/xorg_server.rb:60:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

let me change those constants..

Reply:
There...